### ISSUE TITLE
Empty lines in component/instance options

### ISSUE THREAD

**Issue by rrd108** [rrd108/vue-mess-detector#40]

**Comment by David-Pena** [comment]
When a component would be considered "crowded"? 🤔

**Comment by rrd108** [comment]
What about if the line length exceed 125 characters?

**Comment by David-Pena** [comment]
125 chars sounds fine as the limit to consider it crowded

**Comment by rrd108** [comment]
Ok great.

I changed `analyzer.ts` a little bit, so do not forget to update yours before sending a PR

**Comment by David-Pena** [comment]
I just scrolled to the whole list of changes, great work!

**Comment by rrd108** [comment]
> I just scrolled to the whole list of changes, great work!

I hope I did not ruined your current work with it. I finished something much earlier I planned, so I used my extra time on this.

**Comment by David-Pena** [comment]
> I hope I did not ruined your current work with it. I finished something much earlier I planned, so I used my extra time on this.

Not at all, after my last two PRs I have been a bit busy. But soon I'll get back to decrease that issues count haha
